src/pages/Categories/CategoriesBeta.vue

```diff
@@ -15,49 +15,43 @@
 					Find loans by category
 				</h2>
 			</kv-grid>
-			<kv-grid class="tw-grid-cols-12 tw-pt-3 md:tw-pt-4 lg:tw-pt-6">
-				<div class="tw-col-span-12 md:tw-col-span-6">
-					<main-category-tile tile-size="large" />
-				</div>
-				<div class="tw-col-span-12 md:tw-col-span-6">
-					<main-category-tile tile-size="large" />
-				</div>
-				<div class="tw-col-span-12 md:tw-col-span-6">
-					<main-category-tile tile-size="large" />
-				</div>
-				<div class="tw-col-span-12 md:tw-col-span-6">
-					<main-category-tile tile-size="large" />
+			<kv-grid v-if="categories.length > 0" class="tw-grid-cols-12 tw-pt-3 md:tw-pt-4 lg:tw-pt-6">
+				<div v-for="category in categories.slice(0, 4)" :key="category.id"
+					class="tw-col-span-12 md:tw-col-span-6"
+				>
+					<main-category-tile
+						tile-size="large"
+						:category-name="category.name"
+						:category-description="category.description"
+						:category-image="category.retinaImage.url"
+						:number-loans="category.loans.totalCount"
+					/>
 				</div>
 			</kv-grid>
-			<kv-grid class="tw-grid-cols-12">
-				<div class="tw-col-span-12 md:tw-col-span-4">
-					<main-category-tile tile-size="medium" />
-				</div>
-				<div class="tw-col-span-12 md:tw-col-span-4">
-					<main-category-tile tile-size="medium" />
-				</div>
-				<div class="tw-col-span-12 md:tw-col-span-4">
-					<main-category-tile tile-size="medium" />
+			<kv-grid v-if="categories.length > 0" class="tw-grid-cols-12">
+				<div v-for="category in categories.slice(4, 7)" :key="category.id"
```

Comment:
Are we guaranteed that `categories` will have the required length here? Maybe update the `> 0` check in the line above.